sass/edx/_variables.scss

```diff
@@ -214,13 +215,13 @@ $transition-collapse:    height 0.35s ease !default;
 //
 // Font, line-height, and color for body text, headings, and more.
 
-$font-family-sans-serif: -apple-system, BlinkMacSystemFont, "Open Sans", "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol" !default;
+$font-family-sans-serif: "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif !default;
 //$font-family-monospace:  "SFMono-Regular", Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace !default;
-//$font-family-base:       $font-family-sans-serif !default;
+$font-family-base:       $font-family-sans-serif !default;
 
-$font-size-base: 0.875rem !default; // Assumes the browser default, typically `16px`
-$font-size-lg:   1rem !default;
-$font-size-sm:   0.75rem !default;
+$font-size-base: 1rem !default; // Assumes the browser default, typically `16px`
```

Comment:
Reviewers: I'm not sure about this one, but it made my page better by default. We should decide whether 16px is the right base.